src/package.xml

```diff
@@ -944,9 +964,16 @@
         <members>Trigger_Handler__c.All_Main_Fields</members>
         <name>ListView</name>
     </types>
+    <types>
```

Comment:
@kdy618 I think you mistakenly added this to the package.xml.

Reply:
@robertbwatson , apologies, are you referring to specifically, the custom label that starts from MultipleCourseConnection to OnlyCourseOfferingPopulated?

Reply:
@kdy618 Sorry, I'm referring to lines 967-970 here where the Admin profile got added into the package.xml.